_data/navbar.yml

```diff
@@ -3,6 +3,10 @@
   link: /blog/
   xs: true
 
+- label: Gitter&nbsp;Room
```

Comment:
A two word long menu item looks somewhat out of place since every other one is one word long, would you consider changing it to just `Gitter`?